lms/static/sass/shared-v2/_modal.scss

```diff
@@ -11,261 +11,261 @@
-  color: $base-font-color;
-
-  .inner-wrapper {
+    @include span(5);
     z-index: z-index(mid-front);
-    background: $modal-bg-color;
-    border-radius: 0;
-    border: 1px solid $lms-border-color;
-    box-shadow: inset 0 1px 0 0 rgba(255, 255, 255, 0.7);
-    overflow: hidden;
-    padding-left: ($baseline/2);
-    padding-right: ($baseline/2);
-    padding-bottom: ($baseline/2);
-    position: relative;
-
-    p {
-      font-size: font-size(small);
-      line-height: 1.4;
-    }
-
-    a {
-      &:hover,
-      &:focus {
-        text-decoration: underline;
-      }
-    }
-
-    header {
-      z-index: z-index(mid-front);
-      margin-bottom: ($baseline*1.5);
-      overflow: hidden;
-      padding: 28px $baseline 0;
-      position: relative;
-
-      &::before {
-        @include background-image(radial-gradient(50% 50%, circle closest-side, rgba(255,255,255, 0.8) 0%, rgba(255,255,255, 0) 100%));
-        content: "";
-        display: block;
-        height: 400px;
-        left: 0;
-        margin: 0 auto;
-        position: absolute;
-        top: -140px;
-        width: 100%;
-        z-index: 1;
-      }
-
-      hr {
-        @include background-image(linear-gradient(180deg, rgba(255,255,255, 0) 0%,
-                                                          rgba(255,255,255, 0.8) 50%,
-                                                          rgba(255,255,255, 0)));
-        height: 1px;
-        width: 100%;
-        border: none;
-        margin: 0;
+    display: none;
+    position: absolute;
+    left: 50%;
+    padding: 8px;
+    border-radius: 3px;
+    box-shadow: 0 0 5px 0 $shadow-d1;
+    background: $gray-d2;
+    color: $base-font-color;
+
+    .inner-wrapper {
+        z-index: z-index(mid-front);
+        background: $modal-bg-color;
+        border-radius: 0;
+        border: 1px solid $lms-border-color;
+        box-shadow: inset 0 1px 0 0 rgba(255, 255, 255, 0.7);
+        overflow: hidden;
+        padding-left: ($baseline/2);
+        padding-right: ($baseline/2);
+        padding-bottom: ($baseline/2);
         position: relative;
-        z-index: 2;
-
-        &::after {
-          @include background-image(linear-gradient(180deg, rgba(200,200,200, 0) 0%,
-                                                            rgba(200,200,200, 1) 50%,
-                                                            rgba(200,200,200, 0)));
-          height: 1px;
-          width: 100%;
-          bottom: 0;
-          content: '';
-          display: block;
-          position: absolute;
-          top: -1px;
-        }
-      }
 
-      h2 {
-        @extend .hd-4;
-        position: relative;
-        text-align: center;
-        text-shadow: 0 1px rgba(255,255,255, 0.4);
-        z-index: 2;
-        text-transform: uppercase;
-        font-family: $font-family-serif;
-
-        .edx {
-          text-transform: none;
+        p {
+            font-size: font-size(small);
+            line-height: 1.4;
```

Comment:
Don't we have a line-height variable as well? `spacing-vertical` I think is what we've been using.